qa/c8-orchestration-cluster-e2e-test-suite/v2-stateless-tests/request-validation-test-generator/cache/rest-api.yaml

```diff
@@ -10219,6 +10219,16 @@ components:
         worker:
           description: The name of the worker of this job.
           type: string
+        createdAt:
+          description: When the job was created.
+          type: string
+          format: date-time
+          nullable: true
```

Comment:
💭 We don't use `nullable` for other attributes in this endpoint. @nicpuppa, do we have any guidance yet on using "nullable" and "required" in the spec? If I'm not mistaken, we either list fields as "required" or they can be missing (having no value). We don't transport fields when they are `null` (regardless of whether they are not initialized or explicitly assigned a `null` value). Thus, I believe `nullable` isn't really helpful/meaningful for our API currently.

Reply:
> We don't use nullable for other attributes in this endpoint. 

Sorry, I feel like I am not getting this statement. We have `deniedReason`, `deadline` that are `nullable: true` in this endpoint. Am I missing something?

Reply:
Ha, completely missed those, sorry 😆
I'd still vote against using `nullable` here, though. The other usages are (most probably) misconceptions of how to mark a field as optional. If the attribute can be absent from a response, it should not be in the `required` array. That's it. 

Making it `nullable` means that if it's in the response, it can be a string or have no value. That's not true for our API, though. If it has no value, our API won't return the attribute at this time (we don't return null-value attributes).